myia/debug/inject.py

```diff
@@ -2,7 +2,7 @@
 
 import sys
 import pdb
-from myia.debug.buche import buche
+from .buche import buche
 from buche.debug import BucheDb
 from buche import reader
```

Comment:
To maximize legibility, can we order imports with builtins first, then external dependencies, then current package?
I can't remember if that was in pep8 or another list of good practices.

Reply:
Yes. That's what I do in the other files, but I suppose I forgot to do it there.

Reply:
We can use isort to do it automatically.  There is even a flake8 plugin to check this.

https://pypi.org/project/isort/

Reply:
Well, that's useful. `isort -ca -m 2 myia/**.py` seems good.